opre_ops/ops_site/models.py

```diff
@@ -28,5 +31,43 @@ class Meta:
     def display_name(self):
         return self.first_name + " " + self.last_name
     display_name.short_description = "Full name"
-    
     full_name = property(display_name)
+
+    def __str__(self):
+        return self.full_name
+
+
+ARRANGEMENT_TYPES = [("1", "OPRE Appropriation"), ("2", "Cost Share"), ("3", "IAA"), 
+                    ("4", "IDDA"), ("5", "MOU")]
+class CANInfo(models.Model):
```

Comment:
(nitpick) a newline here.  Although we haven't set up circleci, I wonder if we want to run flake8 before we merge in a pull request manually.  I don't think there's any bad styling, just that it may snowball into a bigger task if we haven't done it since the beginning.